BunqSdk/Exception/ExceptionFactory.cs

```diff
@@ -18,12 +18,12 @@ public class ExceptionFactory
         /// <summary>
         /// String format constants.
         /// </summary>
-        private static string FORMAT_ERROR_MESSAGE = "Response id to help bunq debug: {0}. \n Error message: {1}";
+        private const string FORMAT_ERROR_MESSAGE = "Response id to help bunq debug: {0}. \n Error message: {1}";
         
         /// <returns>The exception that belongs to this status code.</returns>
         public static ApiException CreateExceptionForResponse(
             int responseCode,
-            IList<string> messages,
+            IEnumerable<string> messages,
             string responseId
         )
         {
```

Comment:
Shouldn't this open brace go on the same line as the close parenthese?

Reply:
it is ? 😁